release/ray_release/buildkite/concurrency.py

```diff
@@ -27,10 +28,11 @@
 gpu_cpu_to_concurrency_groups = [
     Condition(min_gpu=9, max_gpu=-1, min_cpu=0, max_cpu=-1, group="large-gpu"),
     Condition(min_gpu=1, max_gpu=9, min_cpu=0, max_cpu=-128, group="small-gpu"),
-    Condition(min_gpu=0, max_gpu=0, min_cpu=513, max_cpu=-1, group="large"),
+    Condition(min_gpu=0, max_gpu=0, min_cpu=1025, max_cpu=-1, group="enormous"),
+    Condition(min_gpu=0, max_gpu=0, min_cpu=513, max_cpu=1024, group="large"),
     Condition(min_gpu=0, max_gpu=0, min_cpu=129, max_cpu=512, group="medium"),
+    Condition(min_gpu=0, max_gpu=0, min_cpu=33, max_cpu=128, group="small"),
     Condition(min_gpu=0, max_gpu=0, min_cpu=0, max_cpu=32, group="tiny"),
-    Condition(min_gpu=0, max_gpu=0, min_cpu=0, max_cpu=128, group="small"),
```

Comment:
This should actually stay here so "small" is the fallback (and not tiny)

Reply:
(maybe add a comment for future reference)

Reply:
(this is needed if a test's resource usage cannot be determined)

Reply:
Added a comment.